specification/flag-evaluation/hooks.md

```diff
@@ -9,28 +9,29 @@ to flag evaluation. They operate similarly to middleware in many web frameworks.
 
 **Hook**: Application author/integrator-supplied logic that is called by the
 OpenFeature framework at a specific stage. **Stage**: An explicit portion of the
-flag evaluation lifecycle. e.g. `before` being "before the evaluation is run.
-**Invocation**: A single call to evaluate a flag. `client.getBooleanValue(..)`
-is an invocation. **API**: The global API singleton.
+flag evaluation lifecycle. e.g. `before` being "before the
+[resolution](../glossary.md#resolving-flag-values) is run. **Invocation**: A
```

Comment:
Updated all cases where we meant "resolution" (getting value from a provider) but said "evaluation" (overall process of getting a value, including resolution, hooks, etc).